assignments/array-methods.js

```diff
@@ -56,22 +56,38 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(currentvalue){
```

Comment:
maybe use 'runner' here instead of currentvalue to be more descriptive.  Use camel case when using 2 or more words currentvalue => currentValue